From a6db45d99e6b776c39823366c6b02a6dec8c4241 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 17 Apr 2014 16:43:57 +0000
Subject: [PATCH 153/462] MANTIS 0031232 change CUSS*.jar: add version
 information to MANIFEST.MF

- set includeantruntime="false" in javac for ant 1.9

git-svn-id: svn://localhost/SelfServiceBuild/trunk@223 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntTargets.xml | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/SelfServiceBuild/Build/java/AntTargets.xml b/SelfServiceBuild/Build/java/AntTargets.xml
index 759efbe..fc158c4 100644
--- a/SelfServiceBuild/Build/java/AntTargets.xml
+++ b/SelfServiceBuild/Build/java/AntTargets.xml
@@ -148,7 +148,8 @@
            target="1.7"
            fork="true"
            memoryinitialsize="256m"
-           memorymaximumsize="512m">
+           memorymaximumsize="512m"
+           includeantruntime="false">
         <classpath refid="project.classpath"/>
         <src path="${GenSrcDir}"/>
         <src path="${SrcDir}"/>
@@ -310,7 +311,8 @@
            deprecation="${javac.deprecation}"
            debug="${javac.debug}"
            verbose="${javac.verbose}"
-           target="1.7">
+           target="1.7"
+           includeantruntime="false">
         <classpath>
           <path refid="testsource.classpath"/>
           <pathelement location="${JarName}"/>
-- 
2.41.0.windows.1

